modules/lb-vshn-hiera/variables.tf

```
@@ -0,0 +1,55 @@
+variable "router_ip_addresses" {
+  type        = list(string)
+  description = "Private IPV4 addresses of nodes running ingress routers"
+}
+
+variable "bootstrap_node" {
+  type        = string
+  description = "The bootstrap nodes private IPV4 adsress"
+  default     = ""
+}
+
+variable "node_name_suffix" {
+  type        = string
+  description = "Suffix to use for node names"
+}
+
+variable "cluster_id" {
+  type        = string
+  description = "ID of the cluster"
+}
+
+variable "privnet_cidr" {
+  default     = "172.18.200.0/24"
+  description = "CIDR of the private net to use"
+}
```

Replace with the following variable, since we don't always have a privnet_cidr for Exoscale

```suggestion
variable "internal_vip" {
  type        = string
  description = "Virtual IP for the Kubernetes/OpenShift API in the internal network
  default     = ""
}
```